### PR TITLE
chore(lint): re-format modules/@angular

### PR DESCRIPTION
**Pull request by alexeagle**
**Please check if the PR fulfills these requirements**
- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit-message-format
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Other... Please describe:

**What is the current behavior?** (You can also link to an open issue here)

**What is the new behavior?**

**Does this PR introduce a breaking change?**
- [ ] Yes
- [ ] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: modules/@angular/core/index.ts [hunk 1]
```diff
@@ -6,23 +6,8 @@
...
-  PlatformRef,
-  ApplicationRef
-} from './src/application_ref';
-export {
-  APP_ID,
-  APP_INITIALIZER,
-  PACKAGE_ROOT_URL,
-  PLATFORM_INITIALIZER
-} from './src/application_tokens';
+export {createPlatform, assertPlatform, disposePlatform, getPlatform, coreBootstrap, coreLoadAndBootstrap, createNgZone, PlatformRef, ApplicationRef} from './src/application_ref';
```
Comment (alexeagle) [inline modules/@angular/core/index.ts]: @mprobst it seems wrong that this line is so long, it should wrap to one per line after hitting 100c right?

Comment (mprobst) [inline modules/@angular/core/index.ts]: This is on purpose – people found that formatting imports is not useful overall, it's just something automatic that your editor should do, so there's no point in formatting them.

If we feel strongly, we could make this an option.

File: modules/@angular/core/src/animation/metadata.ts [hunk 2]
```diff
@@ -211,13 +236,15 @@ export function sequence(steps: AnimationMetadata[]): AnimationSequenceMetadata
 }
 
 /**
- * `style` is an animation-specific function that is designed to be used inside of Angular2's animation
+ * `style` is an animation-specific function that is designed to be used inside of Angular2's
```
Comment (mprobst) [inline modules/@angular/core/src/animation/metadata.ts]: I wonder why we have the wraps here in the comments - did anything change to max line length, or did this just snuck in while `clang-format` was out?

File: modules/@angular/core/test/animation/animation_integration_spec.ts [hunk 3]
```diff
@@ -355,55 +334,84 @@ function declareTests() {
...
-                       expect(keyframes[0]).toEqual([0, {'width': 0}]);
-                       expect(keyframes[1]).toEqual([0.25, {'width': 100}]);
-                       expect(keyframes[2]).toEqual([0.75, {'width': 200}]);
-                       expect(keyframes[3]).toEqual([1, {'width': 300}]);
-                     });
-               })));
+        it(
+            'should create an animation step with multiple keyframes',
+            inject(
+                [TestComponentBuilder, AnimationDriver], fakeAsync(
```
Comment (mprobst) [inline modules/@angular/core/test/animation/animation_integration_spec.ts]: argh. I guess the crazy nested functions below are just hard to read, but we should really consider wrapping the arguments to `inject()` differently.

File: modules/@angular/core/src/animation/metadata.ts (hunk 2 above)
Comment (alexeagle) [inline modules/@angular/core/src/animation/metadata.ts]: all of animation was added right after ng-conf while format check was absent :(

File: modules/@angular/core/index.ts (hunk 1 above)
Comment (alexeagle) [inline modules/@angular/core/index.ts]: okay, if this is how it is, that's fine with me, was just surprised

**Comment by alexeagle** [comment]
@mprobst feels strongly that we should change the approach to only require formatting of changed lines, so I'm going to put this on hold until we discuss more whether/how to do that.

**Comment by mprobst** [comment]
Sorry, that was not what I meant to express :-)

I'm fine with doing this in a big bang, I think just long term we want to
only format changed lines. For stability and also for performance reasons.

File: modules/@angular/core/index.ts (hunk 1 above)
Comment (chuckjaz) [inline modules/@angular/core/index.ts]: I would prefer that if the import exceeds a threshold (such as 100c) then the formatting should change to the one line per member form. This seems to match object literals (although this is a destructuring, not a literal).

File: modules/@angular/common/src/forms-deprecated/model.ts [hunk 4]
```diff
@@ -1,48 +1,49 @@
...
  */
-export const PENDING = "PENDING";
+export const PENDING = 'PENDING';
 
 export function isControl(control: Object): boolean {
   return control instanceof AbstractControl;
 }
 
-function _find(control: AbstractControl, path: Array<string | number>| string) {
+function _find(control: AbstractControl, path: Array<string|number>| string) {
```
Comment (alexeagle) [inline modules/@angular/common/src/forms-deprecated/model.ts]: @mprobst interesting there's a space after the second pipe

File: modules/@angular/common/testing/mock_location_strategy.ts [hunk 5]
```diff
@@ -1,6 +1,8 @@
 import {Injectable} from '@angular/core';
-import {EventEmitter, ObservableWrapper} from '../src/facade/async';
+
```
Comment (alexeagle) [inline modules/@angular/common/testing/mock_location_strategy.ts]: @mprobst the newline seems left behind after the import moved. Seems like we shouldn't have any empty lines in the imports after

Comment (mprobst) [inline modules/@angular/common/testing/mock_location_strategy.ts]: This is intentional, we split absolute URL imports from `'../'` parent path imports (and then ./ imports followed by exports).

Comment (mprobst) [inline modules/@angular/common/testing/mock_location_strategy.ts]: @chuck: I filed a bug for this, we'll make it a setting.

Alex Eagle <notifications@github.com> schrieb am Mi., 8. Juni 2016 um
15:48 Uhr:

File: modules/@angular/core/src/animation/animation_style_util.ts [hunk 6]
```diff
@@ -1,29 +1,32 @@
...
+      });
 
   return finalStyles;
 }
 
-export function balanceAnimationKeyframes(collectedStyles: {[key: string]: string|number},
-                        finalStateStyles: {[key: string]: string|number},
-                        keyframes: any[]): any[] {
+export function balanceAnimationKeyframes(
+    collectedStyles: {[key: string]: string | number},
```
Comment (alexeagle) [inline modules/@angular/core/src/animation/animation_style_util.ts]: @mprobst seems like a bug - spaces appear around pipe inside index signature

Comment (mprobst) [inline modules/@angular/core/src/animation/animation_style_util.ts]: Re spaces around the pipe, indeed, filed an internal issue for that.

Alex Eagle <notifications@github.com> schrieb am Mi., 8. Juni 2016 um
17:19 Uhr:

File: modules/@angular/compiler/private_export.ts [hunk 7]
```diff
@@ -1,95 +1,95 @@
...
+import * as dom_element_schema_registry from './src/schema/dom_element_schema_registry';
+import * as selector from './src/selector';
+import * as style_compiler from './src/style_compiler';
+import * as template_parser from './src/template_parser';
+import * as view_compiler from './src/view_compiler/view_compiler';
 
 export namespace __compiler_private__ {
-  export type SelectorMatcher = selector.SelectorMatcher;
-  export var SelectorMatcher = selector.SelectorMatcher;
+export type SelectorMatcher = selector.SelectorMatcher;
```
Comment (alexeagle) [inline modules/@angular/compiler/private_export.ts]: un-indent here looks like a bug, @mprobst

Comment (mprobst) [inline modules/@angular/compiler/private_export.ts]: @Alex would you mind filing one internally?

Alex Eagle <notifications@github.com> schrieb am Mi., 8. Juni 2016 um
17:35 Uhr:

**Comment by vicb** [comment]
@alexeagle can this PR be closed ?

**Comment by alexeagle** [comment]
obsolete.

**Pull request closed without merging**